mappings/net/minecraft/client/input/KeyCombo.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_4324 net/minecraft/client/input/KeyCombo
```

Comment:
KeyCombo? Not KeyCombination? It's also made out of chars...

Reply:
It's a Mojang name. See the class' `toString`.